scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml

```diff
@@ -571,6 +571,55 @@ WDFN-8-1EP_3x2mm_P0.5mm_EP1.3x1.4mm:
   #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
   #include_suffix_in_3dpath: 'False'
 
+WDFN-8-1EP_4x3mm_P0.65mm_EP2.5x1.9mm:
+  device_type: 'WDFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://www.onsemi.com/pub/Collateral/509AF.PDF'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  body_size_x:
+    nominal: 4.0
+    tolerance: 0
+  body_size_y:
+    nominal: 3.0
+    tolerance: 0
+  lead_width_min: 0.2
+  lead_width_max: 0.3
+  lead_len_min: 0.45
+  lead_len_max: 0.55
+
+  EP_size_x:
+    minimum: 2.3
+    nominal: 2.5
+  EP_size_y:
+    minimum: 1.7
+    nominal: 1.9
```

Comment:
Not sure what you're trying to accomplish.
In either case, it must be `maximum: 2.5` and `maximum: 1.9`, respectively.
If you want to create a 2.4x1.8mm EP as in the original commit, you're done then.
If you actually want 2.5x1.9mm, please use `EP_size_[xy]_overwrite`.